lib/browser/index.js

```diff
@@ -71,6 +71,8 @@ export default class Realm {
 
         [
             'path',
+            'readOnly',
```

Comment:
I understand, but I think we should do what feels right for our binding. If you feel strongly we should keep it this way, then that's fine.